go.mod

```diff
@@ -26,11 +27,13 @@ require (
 	github.com/stretchr/testify v1.6.1
 	github.com/tendermint/btcd v0.1.1
 	github.com/tendermint/crypto v0.0.0-20191022145703-50d29ede1e15
-	github.com/tendermint/go-amino v0.15.1
-	github.com/tendermint/iavl v0.14.0
-	github.com/tendermint/tendermint v0.33.7
+	github.com/tendermint/go-amino v0.15.0
+	github.com/tendermint/iavl v0.14.1
+	github.com/tendermint/tendermint v0.33.8
```

Comment:
What changes does v0.33.8 introduce vs 0.33.7? This change should be documented better in this PR

Reply:
By looking at [Tendermint v0.33.8 CHANGELOG](https://github.com/tendermint/tendermint/blob/v0.33/CHANGELOG.md#v0338), we can see that: 

> Go reported a security vulnerability that affected the encoding/binary package. The most recent binary for tendermint is using 1.14.6, for this reason the Tendermint engineering team has opted to conduct a release to aid users in using the correct version of Go. Read more about the security issue [here](https://github.com/golang/go/issues/40618).

I think we should keep it as v0.33.8 for this reason

Reply:
Makes sense. I'd still prefer to have this change introduced and documented in a separate PR - for the sake of better change management

Reply:
Makes sense, I've reverted this